<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowProperties</class>
 <widget class="QWidget" name="WindowProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>45</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>45</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Hints</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QRadioButton" name="toolType">
      <property name="text">
       <string>Tool</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="windowType">
      <property name="text">
       <string>Window</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="sheetType">
      <property name="text">
       <string>Sheet</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="dialogType">
      <property name="text">
       <string>Dialog</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="drawerType">
      <property name="text">
       <string>Drawer</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="popupType">
      <property name="text">
       <string>Popup</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="splashType">
      <property name="text">
       <string>Splash screen</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QRadioButton" name="tooltipType">
      <property name="text">
       <string>Tooltip</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>54</y>
     <width>313</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QCheckBox" name="MSFSDBox">
      <property name="text">
       <string>MS Windows fixed size dialog</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="minimizeBox">
      <property name="text">
       <string>Window Minimize</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="X11BypassBox">
      <property name="text">
       <string>X11 bypass window manager</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="maximizeBox">
      <property name="text">
       <string>Window Maximize</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="FramelessBox">
      <property name="text">
       <string>Frameless window</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="closeBox">
      <property name="text">
       <string>Window Close</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="NoDropBox">
      <property name="text">
       <string>No drop shadow</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="contextBox">
      <property name="text">
       <string>Window context</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="titleBox">
      <property name="text">
       <string>Window title</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="shadeBox">
      <property name="text">
       <string>Window Shade</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="SystemMenuBox">
      <property name="text">
       <string>Window system menu</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="onTopBox">
      <property name="text">
       <string>Window Stays on top</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="CustomizeBox">
      <property name="text">
       <string>Customize window</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="onBottomBox">
      <property name="text">
       <string>Window Stays on bottom</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
